variables.tf

```diff
@@ -6,6 +6,7 @@ variable "broker_name" {
 variable "subnet_ids" {
   type        = list(string)
   description = "List of VPC subnet IDs"
+  default     = []
```

Comment:
This can't be optional, `aws_lb` resource requires subnets. 
```
subnets            = var.subnet_ids
```

Reply:
`subnets` is optional for `aws_lb` as per [official documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb#subnets).